### ISSUE TITLE
splash screen is not really full screen on iphone x

### ISSUE THREAD

**Issue by ethanyuwang** [expo/expo#1249]
![splash](https://user-images.githubusercontent.com/17556945/34901360-2a74adaa-f7be-11e7-8fbe-a74ba04f92b8.jpeg)
As the image shown the splash picture is not really full screen on iPhone x. I have set the resizeMode to "cover": 

```
{
  "expo": {
    "name": "Glever",
    "orientation": "portrait",
    "sdkVersion": "24.0.0",
    "splash": {
    	"image": "./assets/images/Splash.png",
      "resizeMode": "cover",
    	"backgroundColor": "#378AE4"
	},
	"assetBundlePatterns": [
		"assets/images/*",
		"assets/icons/*"
	],
  }
}

```

my splash image: 

![splash](https://user-images.githubusercontent.com/17556945/34901418-e940f388-f7be-11e7-964d-8875c54d5817.png)

**Comment by terribleben** [comment]
Can you show us what `assets/images/Splash.png` looks like by itself? It's not possible to tell from your screenshot alone whether it's behaving correctly.

**Comment by ethanyuwang** [comment]
@terribleben  Hi I have just uploaded the image itself. the background color is very similar too it so its a bit hard to tell

**Comment by terribleben** [comment]
I think this is the correct behavior, what part of it looks wrong to you?

**Comment by ethanyuwang** [comment]
shouldn't the status bar area be covered with the splash image underneath as well? That's the behavior I see on an iphone 5 and pixel.

**Comment by ethanyuwang** [comment]
weird after a few times of refreshing it works fine now.

**Comment by ethanyuwang** [comment]
![image](https://user-images.githubusercontent.com/17556945/35353466-1fa1c376-00fc-11e8-93ae-ad81e6603079.png)
on the standalone app it has white borders again

**Comment by VeXell** [comment]
Same problem. White paddings on top and bottom after exp build:ios

**Comment by terribleben** [comment]
Tracking in https://github.com/expo/expo/issues/1322

**Comment by terribleben** [comment]
Fixed (#1322), sorry for the disruption here.
